clients/src/main/resources/common/message/DescribeConfigsResponse.json

```diff
@@ -59,7 +59,11 @@
             "about": "The synonym value." },
           { "name": "Source", "type": "int8", "versions": "1+",
             "about": "The synonym source." }
-        ]}
+        ]},
+        { "name": "ConfigValueType", "type": "int8", "versions": "3+", "default": "0", "ignorable": true,
+          "about": "The configuration data type. Type can be one of the following values - BOOLEAN, STRING, INT, SHORT, LONG, DOUBLE, LIST, CLASS, PASSWORD" },
+        { "name": "Documentation", "type": "string", "versions": "3+", "nullableVersions": "0+",
```

Comment:
Why is ConfigValue Type `ignorable`, but `Documentation` not?

Reply:
`Documentation` should be ignorable as well. I'll update the kip as well.